python/cudf/groupby/groupby.py

```diff
@@ -1,21 +1,13 @@
 # Copyright (c) 2018, NVIDIA CORPORATION.
 
-import numpy as np
-import collections
-
 from numbers import Number
-from pandas.api.types import is_categorical_dtype
 
 from cudf.dataframe.dataframe import DataFrame
 from cudf.dataframe.series import Series
-from cudf.dataframe.buffer import Buffer
-from cudf.dataframe.categorical import CategoricalColumn
-from cudf.utils.cudautils import zeros
-from cudf.bindings.nvtx import nvtx_range_pop
-import cudf.dataframe.index as index
-
-from libgdf_cffi import ffi, libgdf
-from librmm_cffi import librmm as rmm
+from cudf.bindings.groupby import (
+    agg as cpp_agg,
```

Comment:
Any reason to have underscode in `_cpp_apply_basic_agg` but not in `cpp_agg`?

Reply:
I just copied what existed, previously. I wasn't trying to remake the wheel 😄.